examples/oci/deploy-all.sh

```diff
@@ -40,26 +51,34 @@ else
   echo "📁 Jaeger Helm Charts already exist. Skipping clone."
 fi
 
-# Navigate into examples/oci if not already in it
-if [[ "$(basename $PWD)" != "oci" ]]; then
-  if [ -d "./examples/oci" ]; then
-    echo "📂 Changing to ./examples/oci directory..."
-    cd ./examples/oci
-  else
-    echo "❌ Cannot find ./examples/oci directory. Exiting."
-    exit 1
-  fi
+# Set image repositories and deploy based on mode
+if [[ "$MODE" == "local" ]]; then
+  echo "🟣 Deploying Jaeger with local registry images..."
+  helm $HELM_JAEGER_CMD jaeger ./helm-charts/charts/jaeger \
+    --set provisionDataStore.cassandra=false \
+    --set allInOne.enabled=true \
+    --set storage.type=memory \
+    --set allInOne.image.repository="cr.jaegertracing.io/jaegertracing/all-in-one" \
+    --set allInOne.image.tag="latest" \
+    --set hotrod.enabled=true \
+    --set hotrod.image.repository="cr.jaegertracing.io/jaegertracing/example-hotrod" \
```

Comment:
if "local" is what you run in the CI, the it should be using local repository

see how docker compose mode does it:
```
  JAEGER_VERSION=$GITHUB_SHA HOTROD_VERSION=$GITHUB_SHA REGISTRY="localhost:5000/" docker compose -f "$docker_compose_file" up -d
```